FV3/coarse_graining/coarse_graining.F90

```diff
@@ -349,12 +359,12 @@ end subroutine block_upsample_2d
 
   subroutine block_upsample_3d(coarse, fine, nz)
     integer, intent(in) :: nz
-    real, intent(in) :: coarse(is_coarse:ie_coarse,js_coarse:je_coarse,1:nz+1)
-    real, intent(out) :: fine(is:ie,js:je,1:nz+1)
+    real, intent(in) :: coarse(is_coarse:ie_coarse,js_coarse:je_coarse,1:nz)
+    real, intent(out) :: fine(is:ie,js:je,1:nz)
 
     integer :: k
 
-    do k = 1, nz + 1
```

Comment:
This is the same issue I referenced here: https://github.com/VulcanClimateModeling/fv3gfs-fortran/pull/58.  Depends which PR is merged first.